apps/api/routers/diagnostics.py

```
@@ -0,0 +1,180 @@
+    except Exception:
+        return False
+
+
+def _cookie_preview(cookie: str) -> str:
+    """Return first 8 chars + '...' as a preview (no ellipsis if not truncated)."""
+    return cookie[:8] + ("..." if len(cookie) > 8 else "")
+
+
+@router.get("/javdb-session", response_model=JavdbSessionStatus)
+def get_javdb_session_status(
+    _user: Dict[str, Any] = Depends(_require_auth),
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)